src/constants/config.json

```diff
@@ -4485,6 +4498,305 @@
 				}
 			]
 		},
+		{
+			"sourceRoute": "/samiksha/v1/solutions/create",
+			"type": "POST",
+			"priority": "MUST_HAVE",
+			"inSequence": false,
+			"orchestrated": false,
+			"targetPackages": [
+				{
+					"basePackageName": "samiksha",
+					"packageName": "elevate-survey"
+				}
+			]
+		},
+		{
+			"sourceRoute": "/samiksha/v1/solutions/details/:id",
+			"type": "POST",
+			"priority": "MUST_HAVE",
+			"inSequence": false,
+			"orchestrated": false,
+			"targetPackages": [
+				{
+					"basePackageName": "samiksha",
+					"packageName": "elevate-survey"
+				}
+			]
+		},
+		{
+			"sourceRoute": "/samiksha/v1/solutions/verifyLink/:id",
+			"type": "POST",
+			"priority": "MUST_HAVE",
+			"inSequence": false,
+			"orchestrated": false,
+			"targetPackages": [
+				{
+					"basePackageName": "samiksha",
+					"packageName": "elevate-survey"
+				}
+			]
+		},
+		{
+			"sourceRoute": "/samiksha/v1/solutions/fetchLink/:id",
+			"type": "GET",
+			"priority": "MUST_HAVE",
+			"inSequence": false,
+			"orchestrated": false,
+			"targetPackages": [
+				{
+					"basePackageName": "samiksha",
+					"packageName": "elevate-survey"
+				}
+			]
+		},
+		{
+			"sourceRoute": "/samiksha/v1/solutions/update/:id",
+			"type": "POST",
+			"priority": "MUST_HAVE",
+			"inSequence": false,
+			"orchestrated": false,
+			"targetPackages": [
+				{
+					"basePackageName": "samiksha",
+					"packageName": "elevate-survey"
+				}
+			]
+		},
+		{
+			"sourceRoute": "/samiksha/v1/solutions/getDetails/:id",
+			"type": "GET",
+			"priority": "MUST_HAVE",
+			"inSequence": false,
+			"orchestrated": false,
+			"targetPackages": [
+				{
+					"basePackageName": "samiksha",
+					"packageName": "elevate-survey"
+				}
+			]
+		},
+		{
+			"sourceRoute": "/samiksha/v1/solutions/list",
+			"type": "GET",
+			"priority": "MUST_HAVE",
+			"inSequence": false,
+			"orchestrated": false,
+			"targetPackages": [
+				{
+					"basePackageName": "samiksha",
+					"packageName": "elevate-survey"
+				}
+			]
+		},
+		{
+			"sourceRoute": "/samiksha/v1/solutions/isTargetedBasedOnUserProfile/:id",
```

Comment:
@praveenKDass if api accept req body the method should be post, please check this for all apis you added

Reply:
Changed as post method @VISHNUDAS-tunerlabs